cobalt/doc/setup-raspi.md

```diff
@@ -0,0 +1,245 @@
+default pi login, and run `sudo raspi-config`. You'll find wifi settings under
+`1. System Options`, then `S1 Wireless LAN`.
+
+## Set up your workstation
+
+The following steps install the cross-compiling toolchain on your workstation.
+The toolchain runs on an x86 workstation and compiles binaries for your ARM
+Raspberry Pi.
+
+1.  Run the following command to install packages needed to build and run
+    Cobalt for Raspberry Pi:
+
+    ```
+    $ sudo apt install -qqy --no-install-recommends g++-multilib \
+        wget xz-utils libxml2 binutils-aarch64-linux-gnu \
+        binutils-arm-linux-gnueabi libglib2.0-dev
+    ```
+
+2.  Choose a location for the installed toolchain &ndash; e.g. `raspi-tools`
+    &ndash; and set `$RASPI_HOME` to that location.
+
+3.  Add `$RASPI_HOME` to your `.bashrc` (or equivalent).
+
+4.  Create the directory for the installed toolchain and go to it:
+
+    ```
+    $ mkdir -p $RASPI_HOME
+    $ cd $RASPI_HOME
+    ```
+
+5.  Download the pre-packaged toolchain and extract it in `$RASPI_HOME`.
+
+    ```
+    $ curl -O https://storage.googleapis.com/cobalt-static-storage-public/cobalt_raspi_tools.tar.bz2
+    $ tar xvpf cobalt_raspi_tools.tar.bz2
+    ```
+
+    (This is a combination of old raspi tools and a newer one from linaro
+     to support older Raspbian Jessie and newer Raspbian Buster)
+
+# Get the Cobalt Code
+
+**This step diverges from Chromium's `fetch` command.** Instead of using `fetch`, please clone the Cobalt repository directly.
+
+```sh
+git clone git@github.com:youtube/cobalt.git cobalt/src
+```
+
+## Configure `gclient`
+
+```sh
+cd cobalt
+gclient config --name=src git@github.com:youtube/cobalt.git
+```
+
+## Download and Sync Sub-repositories
+
+Continue by changing the working directory to `src/`:
+```sh
+cd src
+```
+Further instructions always assume `cobalt/src` as the working directory.
+
+```sh
+gclient sync --no-history -r $(git rev-parse @)
+```
+
+**Note:** This is different from Chromium, the `-r` argument is critical for obtaining the correct version of dependencies.
+
+## Install Build Dependencies
+
+```sh
+./build/install-build-deps.sh
+```
+
+# Build, install, and run Cobalt for Raspberry Pi
+
+##  Congig and Build
+   Navigate to the `cobalt/src` directory and run the following commands
+
+   Configure and Build Cobalt for raspi-2
+   ```
+   $ cobalt/build/gn.py --no-rbe -p evergreen-arm-hardfp-raspi --build_type devel
+   ```
+   Add "use_evergreen = false" to **out/evergreen-arm-hardfp-raspi_devel/args.gn** to turn on Modular
+   ```
+   $ echo "use_evergreen = false" >> out/evergreen-arm-hardfp-raspi_devel/args.gn
+   ```
+   Build cobalt
+   ```
+   $ autoninja -C out/evergreen-arm-hardfp-raspi_devel/ cobalt_loader
+   ```
+## Strip
+   The devel build is 1.2 GB and is too big to load in raspi-2 so just strip it.
+   ```
+   $ cp out/evergreen-arm-hardfp-raspi_devel/libcobalt.so out/evergreen-arm-hardfp-raspi_devel/libcobalt_unstripped.so
```

Comment:
@y4vor is it worth stripping devel builds for raspi (and maybe other platforms) as well? I opted not to when adding the strip tool because I didn't want to remove symbols from tests especially, but maybe it's not particularly helpful?